Breadcrumbs.py

```diff
@@ -108,4 +132,213 @@ def get_points_by_row(row):
               breadcrumbs[index_of_breadcrumb] = breadcrumbs[index_of_breadcrumb][0:length_of_trim]
           break
       number_of_characters_left -= current_length
-    view.set_status('breadcrumbs',separator.join(breadcrumbs+['']))
+  return breadcrumbs
+
+
+def copy(view, text):
+  sublime.set_clipboard(text)
+  sublime.status_message('Breadcrumbs copied to clipboard')
+
+
+class BreadcrumbsEventListener(sublime_plugin.ViewEventListener):
```

Comment:
> so please revert this to the version which used EventListener instead. 

I'll do it so there's both, one enabled for ST2, and a better one for ST3.

> A statusbar which only works in ST3 is not really needed, as in ST3 we now have beautiful phantoms.

I used to think so too, but I've come to really appreciate having this information available at a glance in some languages. I'm all for having options and now that the eventlistener can be effectively disabled there is hardly any cost in keeping that handful of lines of code in there.

> ... which will use toggleable phantoms which follow the cursor

I expect that having phantoms follow your cursor is going to be terrible. The lines are moved apart to make space for a phantom, so if you move your cursor down a line that could be jumpy as the phantom is removed and then recreated a line down. The popup seems like it could be less intrusive, although it will also follow your cursor horizontally and that's super annoying too.  ... I guess you'll have to try it and see.